src/common/tools/gen_listings.ts

```diff
@@ -6,27 +6,26 @@ import { crawl } from './crawl.js';
 
 function usage(rc: number): void {
   console.error('Usage:');
-  console.error('  tools/gen_listings [SUITES...]');
-  console.error('  tools/gen_listings webgpu unittests');
+  console.error('  tools/gen_listings [OUT_DIR] [SRC_DIR] [SUITES...]');
+  console.error('  tools/gen_listings out/ src/ webgpu unittests');
```

Comment:
nit: Could probably just be
```suggestion
  console.error('  tools/gen_listings [OUT_DIR] [PATHS_TO_SUITES...]');
  console.error('  tools/gen_listings out/ src/webgpu src/unittests');
```

Reply:
Done.